bench-vortex/src/engines/ddb/mod.rs

```diff
@@ -44,11 +46,38 @@ pub fn executable_path(user_supplied_path_flag: &Option<path::PathBuf>) -> path:
         }
     }
 
-    let duckdb_path = path::PathBuf::from_str(&format!(
-        "{}/duckdb-vortex/build/release/duckdb",
-        repo_root.unwrap_or_default()
-    ))
-    .expect("failed to create DuckDB executable path");
+    let duckdb_vortex_path = PathBuf::from_str(&repo_root.unwrap_or_else(|| ".".to_string()))
+        .expect("failed to find the vortex repo")
+        .join("duckdb-vortex");
+
+    let mut command = Command::new("make");
+    command
+        .current_dir(&duckdb_vortex_path)
+        .env("GEN", "ninja")
+        .arg("release");
+
+    info!(
+        "Building duckdb vortex extension at {}, with command {:?}",
+        duckdb_vortex_path.display(),
+        command
+    );
+
+    let output = command
+        .output()
+        .expect("Trying to build duckdb vortex extension");
+
+    if !output.status.success() {
+        let stdout = String::from_utf8_lossy(&output.stdout);
+        let stderr = String::from_utf8_lossy(&output.stderr);
+        panic!("duckdb failed: stdout=\"{stdout}\", stderr=\"{stderr}\"");
+    }
+
+    info!(
+        "Built duckdb vortex extension at {}",
+        duckdb_vortex_path.display()
+    );
+
+    let duckdb_path = duckdb_vortex_path.join("build/release/duckdb");
```

Comment:
This PR removes the ability to run system installed duckdb db binaries.

Reply:
We did have that before

Reply:
Ah sorry the GH diff just looked weird, ignore.